deploy-ecs/action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   ecs-service:
     description: ECS service to deploy
     required: true
+  ecs-task-def:
```

Comment:
Can this be `ecs-task-definition`? You can keep the environment variable the same since that's internal, but this is better documentation if it's the full name.

Reply:
Sure, changing it.